CompanyEmployees.Infrastructure.Persistence/Configurations/AttendanceConfiguration.cs

```diff
@@ -0,0 +1,28 @@
+using CompanyEmployees.Core.Domain.Entities;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace CompanyEmployees.Infrastructure.Persistence.Configurations;
+
+public class AttendanceConfiguration : IEntityTypeConfiguration<Attendance>
```

Comment:
Consider explicitly naming the table using `.ToTable("Attendances")` for clarity and to avoid potential pluralization issues with the default naming convention.

**Suggested change**
```suggestion
builder.ToTable("Attendances");
```